src/test/ui/suggestions/suggest-move-types.stderr

```diff
@@ -13,70 +18,123 @@ LL | struct Al<'a, T, M: OneWithLifetime<A=(), T, 'a>> {
    |                                     ----  ^  ^^ generic argument
    |                                     |     |
    |                                     |     generic argument
-   |                                     the first constraint is provided here
+   |                                     the constraint is provided here
+   |
+help: move the constraints after the generic arguments
+   |
+LL | struct Al<'a, T, M: OneWithLifetime<T, 'a, A=()>> {
+   |                                    --    ^^^^
 
 error: generic arguments must come before the first constraint
   --> $DIR/suggest-move-types.rs:40:46
    |
 LL | struct B<T, U, V, M: Three<A=(), B=(), C=(), T, U, V>> {
-   |                            ----              ^  ^  ^ generic argument
-   |                            |                 |  |
-   |                            |                 |  generic argument
-   |                            |                 generic argument
-   |                            the first constraint is provided here
+   |                            ----  ----  ----  ^  ^  ^ generic argument
+   |                            |     |     |     |  |
+   |                            |     |     |     |  generic argument
+   |                            |     |     |     generic argument
+   |                            |     |     the constraints are provided here
+   |                            |     the constraints are provided here
+   |                            the constraints are provided here
```

Comment:
This message seems odd. It's saying several times that a bunch of constraints are located in the highlighted span, but it's just one constraint. *"a constraint is provided here"* would be correct I think, but also overly and unnecessarily verbose.